src/client/tablet_client.h

```diff
@@ -178,18 +178,19 @@ class TabletClient : public Client {
     bool UpdateTTL(uint32_t tid, uint32_t pid, const ::openmldb::type::TTLType& type, uint64_t abs_ttl,
                    uint64_t lat_ttl, const std::string& index_name);
 
-    bool DeleteBinlog(uint32_t tid, uint32_t pid);
+    bool DeleteBinlog(uint32_t tid, uint32_t pid, ::openmldb::common::StorageMode storage_mode);
 
     bool GetTaskStatus(::openmldb::api::TaskStatusResponse& response);  // NOLINT
 
     bool DeleteOPTask(const std::vector<uint64_t>& op_id_vec);
 
     bool GetTermPair(uint32_t tid, uint32_t pid,
+                     ::openmldb::common::StorageMode storage_mode,  // NOLINT
```

Comment:
it is no need to add storge_mode to `GetTermPair`.  it can get from table

Reply:
Yes, but you can call this function even if the table does not exist. And when the table does not exist, storage mode is still needed to get term and snapshot offset.